docs/_extra/api-reference/hypothesis.yaml

```diff
@@ -502,32 +502,58 @@ paths:
           minimum: 0
           maximum: 200
           default: 20
+        - name: sort
+          in: query
+          description: The field by which annotations should be sorted.
+          required: false
+          type: string
+          enum: [created, updated, group, id, user]
+          default: updated
         - name: search_after
           in: query
-          description: Returns results after the annotation who's sort field has this value.
-            If specifying a date use the format yyyy-MM-dd'T'HH:mm:ss.SSX or time in
-            miliseconds since the epoch. This is used for iteration through large collections
-            of results.
+          example: 2019-01-03T19:46:09.334539+00:00
```

Comment:
Is the timezone part (`+00:00`) and fine-grained time (hours, mins, secs, microseconds) required or optional?

Reply:
One thing I wrestled with was how much to expose about the formats that would actually _work_ here. At the validation level, we actually accept anything that can be parsed by the [dateutil](https://pypi.org/project/python-dateutil/) dependency. The parsing therein is permissive, but I feel like saying "any date parse-able by `dateutil`" is really an implementation detail at this point. I think some consideration and discussion might be useful here...

It is my assumption (possibly wrong!) that the most common use of the `search_after` option when dealing in chronology/dates would be to plug in the datetime string from the last annotation of the previous page—a previous page of results also returned by our API. Because our API deals in the ISO8601 format listed in the example, it felt consistent. That is, client code could simply fling back a date string that our API generated in the first place.

My lightly-held opinion at this time is that perhaps we _should_ enforce stricter datetime formats and make our entire API surface always speak the same date/time vernacular. So I kind of met in the middle here and said we accept ISO 8601 dates or timestamps, which is true but technically not the whole gamut of what we do accept.

Oh, to answer your question: it's optional.